proto/job.proto

```diff
@@ -12,11 +12,13 @@ import "google/protobuf/timestamp.proto";
 
 option go_package = "github.com/diagridio/go-etcd-cron/api";
 
-// JobStored is the wrapped stored version of a Job which has a uuid
+// JobStored is the wrapped stored version of a Job which has a partition_id
 // associated.
 message JobStored {
-  // uuid is a unique identifier for the job.
-  uint32 uuid = 1;
+  // partion_id is an identifier for the job, used for distinguishing jobs with
+  // the same name and assigning the job to a partition.
+  // Doesn't need to be globally unique.
+  uint32 partition_id = 1;
```

Comment:
Isn't this just an id for the job? Why add partition to the front of the name? Partition implies something to do with the partitioning of the jobs which is kind of confusing. Or is this used for keeping track of which 'partition' or scheduler owns which job?

Reply:
We could change to just `id` however, it needs to be clear that this is not a globally unique ID for the job (that's the job name), and doesn't matter if there are conflicts. It is used to partition the job to a particular scheduler, and have a staleness check for whether a counter matches this job object.